source/includes/2016-03-05/_disputes.md

```diff
@@ -1162,6 +1162,67 @@ chargehound.disputes.submit("dp_123",
 
 In order to submit a Braintree dispute, you will also need to attach the Braintree transaction id using the `charge` parameter. 
 
+## Stripe listing needs response
+
+> Example usage:
+
+```sh
+curl https://api.chargehound.com/v1/disputes?needs_response&warning_needs_response \
```

Comment:
these parameters are not correct. should be `state=needs_response&state=warning_needs_response` I believe